templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base_generic.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+  <h2>Create or Update Manufacturer</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <button type="submit" class="btn btn-primary">Submit</button>
+  </form>
```

Comment:
According to the checklist, you should add a `Cancel` button that leads to the previous page the user was on. This is important for the user experience, allowing users to easily navigate away from the form without submitting it.